travis/minimesos

```diff
@@ -2,9 +2,9 @@
 
 set -e
 
-MINIMESOS_TAG="0.13.0"
-PARAMS="$@"
-MINIMESOS_CLI_IMAGE="containersol/minimesos-cli"
+MINIMESOS_TAG="0.13.1"
+PARAMS="--debug $@"
+MINIMESOS_CLI_IMAGE="daowen/minimesos-cli"
```

Comment:
Using a custom patched Minimesos makes it harder for me to play with different Mesos versions that Minimesos offers. I'm not sure how I feel about this change :(

Reply:
I don't think that's true. This is the same version we've been using, plus the patch for [this bug](https://github.com/ContainerSolutions/minimesos/issues/532). We can ditch this as soon as the minimesos teams makes their next release (which will include the fix that I had to apply manually). This only affects the `minimesos-cli` image, which is just the controller used to start/stop the cluster of docker instances. None of the long-running containers are affected.

Reply:
Does that mean I can play around with different versions by changing the value of `MINIMESOS_TAG`? IIRC, changing that value is how I experimented with different versions of Mesos.

Reply:
No, you wouldn't be able to change the `MINIMESOS_TAG` value—but I don't think that's what controls the Mesos version. Are you sure you weren't editing [the `mesosVersion` number in the minimesosFile?](https://github.com/DaoWen/Cook/blob/4689f3cfb2e7dc9f5c0eaa020aacbbcacbd55818/travis/minimesosFile#L6)

Reply:
Yes! thanks.